pandas/core/generic.py

```diff
@@ -1039,6 +1039,32 @@ def to_clipboard(self, excel=None, sep=None, **kwargs):
         from pandas.io import clipboard
         clipboard.to_clipboard(self, excel=excel, sep=sep, **kwargs)
 
+    def to_xarray(self):
+        """
+        Return an xarray object from the pandas object.
+
+        Returns
+        -------
+        a DataArray for a Series
+        a Dataset for a DataFrame
+        a DataArray for higher dims
+
+        See Also
+        --------
+        `xarray docs <http://xarray.pydata.org/en/stable/>`__
```

Comment:
Possibly it is this line where it is choking on, as in a See also should come a python object

I would just make a 'Note' section of it instead of See also, and then say "See also the xarray docs .."